### PR TITLE
feat: add VaDropdown keepAnchorHeight property (#3083)

### PR DESCRIPTION
**Pull request by ilearnjs**
Add VaDropdown keepAnchorHeight property 

## Description

Add new property for VaDropdown which fixes the problem when dropdown height is larger than viewport or VaDropdown  target
For more details see video here https://github.com/epicmaxco/vuestic-ui/issues/3083

Return position aliases

## Markup:

<details>

```vue
// Your code
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/va-dropdown/hooks/useDropdown.ts [hunk 1]
```diff
@@ -25,13 +26,13 @@ export const useDropdown = (
...
-    if (options.value.placement === 'auto') {
-      return 'bottom'
-    }
+    const { position, align } = usePlacementAliases({ placement: options.value.placement })
 
-    return options.value.placement
+    return `${position.value}-${align.value}` as Placement
   })
 
+  console.log('placementComputed', placementComputed.value)
```
Comment (ilearnjs) [inline packages/ui/src/components/va-dropdown/hooks/useDropdown.ts:34]:
```suggestion
```

File: packages/ui/src/components/va-dropdown/hooks/useDropdown.ts [hunk 2]
```diff
@@ -25,13 +26,13 @@ export const useDropdown = (
...
+
```
Comment (ilearnjs) [inline packages/ui/src/components/va-dropdown/hooks/useDropdown.ts:35]:
```suggestion
```

**Review by m0ksem — CHANGES_REQUESTED** [review]
Maybe rename prop to `takeAvaialbeHeight`. Or maybe make it as default behavior and name prop like `ignoreAvailableHeight` that will be used to disable this behavior.

You also need to rebase branch with develop and cherry-pick your commits. Because we squash commits there will be duplicated code changed.

File: packages/docs/page-config/ui-elements/dropdown/api-description.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ export default defineApiDescription({
     closeOnContentClick: "Dropdown will be closed when clicked inside dropdown content",
     hoverOutTimeout: "Time in `ms` after mouse leave dropdown before it will be closed",
     hoverOverTimeout: "Time in `ms` after mouse enter dropdown before it will be opened",
+    keepAnchorHeight: "If true, dropdown content will adjust its height when the content is larger than available space",
```
[the remark this answers is not in the document]
Comment (ilearnjs) [reply packages/docs/page-config/ui-elements/dropdown/api-description.ts]: Renamed to `verticalScrollOnOverflow`

**Comment by ilearnjs** [comment]
Moved here https://github.com/epicmaxco/vuestic-ui/pull/3501

**Pull request closed without merging**